sdk/trace/src/main/java/io/opentelemetry/sdk/trace/samplers/ParentBasedSampler.java

```diff
@@ -52,9 +53,9 @@ public SamplingResult shouldSample(
       Context parentContext,
       String traceId,
       String name,
-      Span.Kind spanKind,
-      ReadableAttributes attributes,
-      List<SpanData.Link> parentLinks) {
+      Kind spanKind,
+      Attributes attributes,
+      List<Link> parentLinks) {
```

Comment:
interesting...are we allowing inner-class imports now? [I think it's fine...I just thought this would fail spotless]